orttraining/orttraining/python/training/orttrainer.py

```diff
@@ -635,6 +638,8 @@ def _create_ort_training_session(self):
         ort_parameters.optimizer_int_attributes_map = optimizer_int_attributes_map
         if bool(self._optim_state_dict):
```

Comment:
we may need a plan to move the temporary experimental load_optimizer_state function. It can either be done in this PR or in a follow up one.

Reply:
I'll do that in a follow up one (along with other cleanups).